posthog/migrations/0216_uniqueness_constraint_tagged_items.py

```diff
@@ -0,0 +1,139 @@
+# Generated by Django 3.2.5 on 2022-03-03 17:28
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    atomic = False
+
+    dependencies = [
+        ("posthog", "0215_add_tags_back"),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            sql="""
+                CREATE UNIQUE INDEX CONCURRENTLY "unique_dashboard_tagged_item" ON "posthog_taggeditem" ("tag_id", "dashboard_id")
+                WHERE ("insight_id" IS NULL AND "event_definition_id" IS NULL AND "property_definition_id" IS NULL AND "action_id" IS NULL);
```

Comment:
Maybe I'm missing something, but what is the issue with just specifying `WHERE dashboard_id IS NOT NULL` ?

This would seem less brittle for example when adding another field.

Reply:
You're completely right, sorry if I missed that in the previous review! I ran the tests with this changed to NOT NULL and seems to work the same.